### PR TITLE
Update PPP - Daniel

### PR DESCRIPTION
**Pull request by dandaandaaaaaan**
This PR updates my PPP with more recent contributions.

I think this will change for sure especially once we synchronise the format of how we want to write the content of the PPP (for example, possibly include links to PRs, etc.), so good chance I will update this before it is ready to be merged. 

Live version can be seen [here](https://dandaandaaaaaan.github.io/tp/team/dandaandaaaaaan.html)

### CODE REVIEW

File: docs/team/dandaandaaaaaan.md [hunk 1]
```diff
@@ -4,34 +4,43 @@ title: Daniel's Project Portfolio Page
 ---
 ### Project: ContacX
```
Comment (sharpstorm) [inline docs/team/dandaandaaaaaan.md:5]: Note the spelling of product name is wrong here.

File: docs/team/dandaandaaaaaan.md [hunk 2]
```diff
@@ -4,34 +4,43 @@ title: Daniel's Project Portfolio Page
...
 
 Given below are my contributions to the project.
 
 * **New Feature**: Added the import and export features for `.csv`
     * What it does: Allows the user to import contacts from a Microsoft Excel compatible format, as well as export it back into the same format. Import CSV allows for customising of column numbers for each field.
     * Justification: As our target audience is professionals, it is very common that they will have contacts in Excel in some form. By designing an Import CSV command with flexibility of column numbers, professionals can easily migrate their existing contacts with minimal modifications.
+* **New Feature**: Internal parser to use a markdown-like syntax to style text components
+    * What it does: Allows for styled command execution results, through Bold/Italic/Monospaced text, to make error messages nicer and easier to read. e.g. a String of `**Bold**` will display as **Bold**, similar to in Markdown. 
+    * Justification: In order to provide feedback to users in a more organised manner, especially since error messages of a command can contain many components.
```
Comment (sharpstorm) [inline docs/team/dandaandaaaaaan.md:18]: This sentence is not really fluent, consider rephrasing it.

File: docs/team/dandaandaaaaaan.md [hunk 3]
```diff
@@ -4,34 +4,43 @@ title: Daniel's Project Portfolio Page
...
+    * Functionality, documentation and tests of Markdown-like test styler
+      * Includes components mainly in `UI`
+      
 * **Project management**:
     * Code reviews, workflow establishment
     * Creation of releases, ensuring passing CI
 
 * **Enhancements to existing features**:
     * Modified the constraints for `Phone` to allow country codes within the numbers. It is common that other contact management apps use this, and can cause problems when importing, so we decided to allow it.
+    * Modified all existing error messages to have a specific format. i.e. `monospaced` for the command words, *italics* for parameters, etc.
```
Comment (sharpstorm) [inline docs/team/dandaandaaaaaan.md:33]: I think command usage messages would be a better description of it.

Comment (dandaandaaaaaan) [reply docs/team/dandaandaaaaaan.md]: thanks for the suggestion, implemented in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/316/commits/7702cb4a1d86c0856f6ea62e77ccc7968cd115fc

File: docs/team/dandaandaaaaaan.md (hunk 1 above)
Comment (dandaandaaaaaan) [reply docs/team/dandaandaaaaaan.md]: thanks for the spot, fixed in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/316/commits/7702cb4a1d86c0856f6ea62e77ccc7968cd115fc

File: docs/team/dandaandaaaaaan.md [hunk 4]
```diff
@@ -2,39 +2,47 @@
...
 
 * **New Feature**: Added the import and export features for `.csv`
     * What it does: Allows the user to import contacts from a Microsoft Excel compatible format, as well as export it back into the same format. Import CSV allows for customising of column numbers for each field.
     * Justification: As our target audience is professionals, it is very common that they will have contacts in Excel in some form. By designing an Import CSV command with flexibility of column numbers, professionals can easily migrate their existing contacts with minimal modifications.
+* **New Feature**: Internal parser to style text components with a Markdown-like syntax
+    * What it does: Allows for dynamically generated styled text, through Bold/Italic/Monospaced text, mainly used in command usage messages. e.g. a String of `**Bold**` will display as **Bold**, similar to in Markdown. 
+    * Justification: In order to allow for developers to easily create styled text as feedback to users. Some text components can get very long, especially command usage messages. Through the use of this parser, information can be presented to the users in a more organised manner, making it easier to read.
```
Comment (sharpstorm) [inline docs/team/dandaandaaaaaan.md:18]: I think the weird part is starting with `In order`. Consider just starting the sentence with `To allow ...`.

File: docs/team/dandaandaaaaaan.md [hunk 5]
```diff
@@ -2,39 +2,47 @@
...
-
+    * Functionality, documentation and tests of Markdown-like test styler
+      * Includes components mainly in `UI`
+      
 * **Project management**:
-    * Code reviews, workflow establishment
-    * Creation of releases, ensuring passing CI
+    * Established the overall team workflow.
+    * Ensuring code quality through reviews.
+    * Creation of releases, ensuring CI passing.
```
Comment (sharpstorm) [inline docs/team/dandaandaaaaaan.md:30]: Consider the following phrasing.
```suggestion
    * Creating and Managing the lifecycle of releases, including version number maintainence in code.
    * Ensuring that CI failures are resolved in a timely manner.
```

Comment (dandaandaaaaaan) [reply docs/team/dandaandaaaaaan.md]: thx for suggestion, implemented in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/316/commits/fbd807ab751c170f9493d3a19fc13c8d319cd5f1

File: docs/team/dandaandaaaaaan.md (hunk 4 above)
Comment (dandaandaaaaaan) [reply docs/team/dandaandaaaaaan.md]: thx for suggestion, implemented in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/316/commits/fbd807ab751c170f9493d3a19fc13c8d319cd5f1

**Review by sharpstorm — APPROVED** [review]
LGTM.

**Review by Wei-Xinn — APPROVED** [review]
lgtm

**Pull request merged**